content/en/docs/zero-code/java/spring-boot-starter/annotations.md

```diff
@@ -1,57 +1,85 @@
-span by annotating the method parameters with `@SpanAttribute`.
-
-> **Note**: this annotation can only be applied to bean methods managed by the
-> spring application context. To learn more about aspect weaving in spring, see
-> [spring-aop](https://docs.spring.io/spring/docs/current/spring-framework-reference/core.html#aop).
-
-| Feature     | Property                                   | Default Value | Description                       |
-| ----------- | ------------------------------------------ | ------------- | --------------------------------- |
-| `@WithSpan` | `otel.instrumentation.annotations.enabled` | true          | Enables the WithSpan annotations. |
+If you add the `WithSpan` annotation to a method, the method will be wrapped in a span. The `SpanAttribute` annotation allows you to capture the method arguments as attributes.
 
 ```java
-import org.springframework.stereotype.Component;
-
 import io.opentelemetry.instrumentation.annotations.SpanAttribute;
 import io.opentelemetry.instrumentation.annotations.WithSpan;
-import io.opentelemetry.api.trace.Span;
-import io.opentelemetry.api.trace.SpanKind;
-
-/**
- * Test WithSpan
- */
-@Component
-public class TracedClass {
 
     @WithSpan
-    public void tracedMethod() {
+    public void tracedMethod(@SpanAttribute parameter) {
     }
+```
 
-    @WithSpan(value="span name")
-    public void tracedMethodWithName() {
-        Span currentSpan = Span.current();
-        currentSpan.addEvent("ADD EVENT TO tracedMethodWithName SPAN");
-        currentSpan.setAttribute("isTestAttribute", true);
-    }
+{{% alert title="Note" color="info" %}}
+The OpenTelemetry annotations use Spring AOP based on proxys. 
 
-    @WithSpan(kind = SpanKind.CLIENT)
-    public void tracedClientSpan() {
+These annotations only work for the methods of the proxy. You can learn more in the [Spring documentation](https://docs.spring.io/spring-framework/reference/core/aop/proxying.html).
+
+In the following example, the `WithSpan` annotation won't do anything when the GET endpoint is called:
+
+```java
+@RestController
+public class MyControllerManagedBySpring {
+  
+    @GetMapping("/ping")
+    public void aMethod() {
+        anotherMethod();
     }
 
-    public void tracedMethodWithAttribute(@SpanAttribute("attributeName") String parameter) {
+    @WithSpan
+    public void anotherMethod() {
     }
 }
 ```
+
+{{% /alert %}}
+
+
+{{% alert title="Note" color="info" %}}
+
+{{< tabpane text=true >}} {{% tab header="Maven (`pom.xml`)" lang=Maven %}}
+To be able to use the OpenTelemetry annotations, you have to add the Spring Boot Starter AOP dependency to your project:
+
+```xml
+<dependencies>
+  <dependency>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-aop</artifactId>
+  </dependency>
+</dependencies>
+```
+
+```kotlin
+dependencies {
+  implementation("org.springframework.boot:spring-boot-starter-aop")
+}
+```
```

Comment:
One of the CI failures was caused by a missing tabpane closure. I added that closure along with a second tab, but if this isn't what you intended, please disregard. We should also double-check this renders correctly once the preview is working.

```suggestion
To be able to use the OpenTelemetry annotations, you have to add the Spring Boot Starter AOP dependency to your project:
{{< tabpane text=true >}} 
{{% tab header="Maven (`pom.xml`)" lang=Maven %}}
<dependencies>
  <dependency>
    <groupId>org.springframework.boot</groupId>
    <artifactId>spring-boot-starter-aop</artifactId>
  </dependency>
</dependencies>
{{% /tab %}}
{{% tab header="Kotlin" lang=Kotlin %}}
dependencies {
  implementation("org.springframework.boot:spring-boot-starter-aop")
}
{{% /tab %}}
{{< /tabpane >}}
```